rails-commenting.rb

```diff
@@ -6,25 +6,28 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) The controller method of index is the "R" in CRUD and the http verb associated with index is "GET".
```

Comment:
Make sure to read the directions.  In this case, problem 1 is asking to only describe the following line, which is where class BlogPostsController inherits from ApplicationController and will be where all of the controller methods will be written.